Valentin_Khokhlov/1/koans/about_dice_project.rb

```diff
@@ -0,0 +1,76 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# Implement a DiceSet Class here:
+#
+class DiceSet
+  attr_reader :dice
+
+  def roll(count)
+    @dice = []
+    pool = [*1..6]
+    
+    count.times do
+      @dice << pool.sample
+    end
```

Comment:
If `count` is very long (for example, 1_000_000_000), then it is better to allocate memory on array creation. Like this:
```ruby
@dice = Array.new(count)

...

count.times do |i|
  @dice[i] = pool.sample
end
```

Also, you can write it in one line:
```ruby
 @dice = Array.new(count) { pool.sample }
```